### ISSUE TITLE
Use a bookmark to inject JavaScript

### ISSUE THREAD

**Issue by szmarczak** [NikolaiT/uncaptcha3#1]
The title says it all. No need to open a dev console. The URL is `javascript:alert(1)`.

**Comment by NikolaiT** [comment]
Good idea. Currently I start the browser in remote debug mode and I inject the JavaScript via puppeteer. This works very fine.
